apps/web/src/pages/cambridge/index.tsx

```diff
@@ -33,7 +33,7 @@ const IndexPage = () => (
         ]}
         right={[
           {
-            text: 'Donate',
+            text: 'Get Involved',
             href: 'donate/',
```

Comment:
I think you'd also want to update this link to where they can get involved - e.g. a Google form application/link to a relevant social media post etc.